testutil/utils.go

```diff
@@ -24,10 +24,11 @@ import (
 	"testing"
 	"time"
 
-	"github.com/stretchr/testify/require"
-
 	"github.com/dgraph-io/dgo/v210"
 	"github.com/dgraph-io/dgraph/x"
+	"github.com/dgrijalva/jwt-go"
```

Comment:
same here, import order needs to be changed

Reply:
Addressed.